packages/ui-extensions/src/surfaces/checkout/api/order-identity/order-identity.ts

```diff
@@ -0,0 +1,16 @@
+import {StatefulRemoteSubscribable} from '@remote-ui/async-subscription';
+
+export interface OrderIdentity {
+  id: string;
```

Comment:
We should explain what that id is here (the order id that will be used by the order when created). I think we could probably copy the content from the corresponding GraphQL field.